<html>
<head>
<title>TestNG:  DummyTest</title>
<link href="../testng.css" rel="stylesheet" type="text/css" />
<link href="../my-testng.css" rel="stylesheet" type="text/css" />

<style type="text/css">
.log { display: none;} 
.stack-trace { display: none;} 
</style>
<script type="text/javascript">
<!--
function flip(e) {
  current = e.style.display;
  if (current == 'block') {
    e.style.display = 'none';
    return 0;
  }
  else {
    e.style.display = 'block';
    return 1;
  }
}

function toggleBox(szDivId, elem, msg1, msg2)
{
  var res = -1;  if (document.getElementById) {
    res = flip(document.getElementById(szDivId));
  }
  else if (document.all) {
    // this is the way old msie versions work
    res = flip(document.all[szDivId]);
  }
  if(elem) {
    if(res == 0) elem.innerHTML = msg1; else elem.innerHTML = msg2;
  }

}

function toggleAllBoxes() {
  if (document.getElementsByTagName) {
    d = document.getElementsByTagName('div');
    for (i = 0; i < d.length; i++) {
      if (d[i].className == 'log') {
        flip(d[i]);
      }
    }
  }
}

// -->
</script>

</head>
<body>
<h2 align='center'>DummyTest</h2><table border='1' align="center">
<tr>
<td>Tests passed/Failed/Skipped:</td><td>6/4/0</td>
</tr><tr>
<td>Started on:</td><td>Wed Jul 12 15:51:09 IST 2017</td>
</tr>
<tr><td>Total time:</td><td>0 seconds (122 ms)</td>
</tr><tr>
<td>Included groups:</td><td></td>
</tr><tr>
<td>Excluded groups:</td><td></td>
</tr>
</table><p/>
<small><i>(Hover the method name to see the test class name)</i></small><p/>
<table width='100%' border='1' class='invocation-failed'>
<tr><td colspan='4' align='center'><b>FAILED TESTS</b></td></tr>
<tr><td><b>Test method</b></td>
<td width="30%"><b>Exception</b></td>
<td width="10%"><b>Time (seconds)</b></td>
<td><b>Instance</b></td>
</tr>
<tr>
<td title='tests.TestClass1.TempTest3()'><b>TempTest3</b><br>Test class: tests.TestClass1</td>
<td><div><pre>java.lang.AssertionError: The following asserts failed:
	expected [3] but found [4]
	at assertions.MyAssertion.assertAll(MyAssertion.java:11)
	at tests.TestClass1.TempTest3(TestClass1.java:33)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
... Removed 11 stack frames</pre></div><a href='#' onClick='toggleBox("stack-trace1458592728", this, "Click to show all stack frames", "Click to hide stack frames")'>Click to show all stack frames</a>
<div class='stack-trace' id='stack-trace1458592728'><pre>java.lang.AssertionError: The following asserts failed:
	expected [3] but found [4]
	at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:43)
	at assertions.MyAssertion.assertAll(MyAssertion.java:11)
	at tests.TestClass1.TempTest3(TestClass1.java:33)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:104)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:645)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:851)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1177)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
</pre></div></td>
<td>0</td>
<td>tests.TestClass1@ea4a92b</td></tr>
<tr>
<td title='tests.TestClass1.TempTest4()'><b>TempTest4</b><br>Test class: tests.TestClass1</td>
<td><div><pre>java.lang.AssertionError: The following asserts failed:
	expected [4] but found [5]
	at assertions.MyAssertion.assertAll(MyAssertion.java:11)
	at tests.TestClass1.TempTest4(TestClass1.java:41)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
... Removed 11 stack frames</pre></div><a href='#' onClick='toggleBox("stack-trace892638352", this, "Click to show all stack frames", "Click to hide stack frames")'>Click to show all stack frames</a>
<div class='stack-trace' id='stack-trace892638352'><pre>java.lang.AssertionError: The following asserts failed:
	expected [4] but found [5]
	at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:43)
	at assertions.MyAssertion.assertAll(MyAssertion.java:11)
	at tests.TestClass1.TempTest4(TestClass1.java:41)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:104)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:645)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:851)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1177)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
</pre></div></td>
<td>0</td>
<td>tests.TestClass1@ea4a92b</td></tr>
<tr>
<td title='tests.TestClass2.TempTest8()'><b>TempTest8</b><br>Test class: tests.TestClass2</td>
<td><div><pre>java.lang.AssertionError: The following asserts failed:
	expected [8] but found [9]
	at assertions.MyAssertion.assertAll(MyAssertion.java:11)
	at tests.TestClass2.TempTest8(TestClass2.java:34)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
... Removed 11 stack frames</pre></div><a href='#' onClick='toggleBox("stack-trace1089658171", this, "Click to show all stack frames", "Click to hide stack frames")'>Click to show all stack frames</a>
<div class='stack-trace' id='stack-trace1089658171'><pre>java.lang.AssertionError: The following asserts failed:
	expected [8] but found [9]
	at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:43)
	at assertions.MyAssertion.assertAll(MyAssertion.java:11)
	at tests.TestClass2.TempTest8(TestClass2.java:34)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:104)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:645)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:851)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1177)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
</pre></div></td>
<td>0</td>
<td>tests.TestClass2@3c5a99da</td></tr>
<tr>
<td title='tests.TestClass2.TempTest9()'><b>TempTest9</b><br>Test class: tests.TestClass2</td>
<td><div><pre>java.lang.AssertionError: The following asserts failed:
	expected [9] but found [10]
	at assertions.MyAssertion.assertAll(MyAssertion.java:11)
	at tests.TestClass2.TempTest9(TestClass2.java:42)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
... Removed 11 stack frames</pre></div><a href='#' onClick='toggleBox("stack-trace299569201", this, "Click to show all stack frames", "Click to hide stack frames")'>Click to show all stack frames</a>
<div class='stack-trace' id='stack-trace299569201'><pre>java.lang.AssertionError: The following asserts failed:
	expected [9] but found [10]
	at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:43)
	at assertions.MyAssertion.assertAll(MyAssertion.java:11)
	at tests.TestClass2.TempTest9(TestClass2.java:42)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:104)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:645)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:851)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1177)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
</pre></div></td>
<td>0</td>
<td>tests.TestClass2@3c5a99da</td></tr>
</table><p>
<table width='100%' border='1' class='invocation-passed'>
<tr><td colspan='4' align='center'><b>PASSED TESTS</b></td></tr>
<tr><td><b>Test method</b></td>
<td width="30%"><b>Exception</b></td>
<td width="10%"><b>Time (seconds)</b></td>
<td><b>Instance</b></td>
</tr>
<tr>
<td title='tests.TestClass1.TempTest1()'><b>TempTest1</b><br>Test class: tests.TestClass1</td>
<td></td>
<td>0</td>
<td>tests.TestClass1@ea4a92b</td></tr>
<tr>
<td title='tests.TestClass2.TempTest10()'><b>TempTest10</b><br>Test class: tests.TestClass2</td>
<td></td>
<td>0</td>
<td>tests.TestClass2@3c5a99da</td></tr>
<tr>
<td title='tests.TestClass1.TempTest2()'><b>TempTest2</b><br>Test class: tests.TestClass1</td>
<td></td>
<td>0</td>
<td>tests.TestClass1@ea4a92b</td></tr>
<tr>
<td title='tests.TestClass1.TempTest5()'><b>TempTest5</b><br>Test class: tests.TestClass1</td>
<td></td>
<td>0</td>
<td>tests.TestClass1@ea4a92b</td></tr>
<tr>
<td title='tests.TestClass2.TempTest6()'><b>TempTest6</b><br>Test class: tests.TestClass2</td>
<td></td>
<td>0</td>
<td>tests.TestClass2@3c5a99da</td></tr>
<tr>
<td title='tests.TestClass2.TempTest7()'><b>TempTest7</b><br>Test class: tests.TestClass2</td>
<td></td>
<td>0</td>
<td>tests.TestClass2@3c5a99da</td></tr>
</table><p>
</body>
</html>